apps/web/src/components/avatars/create-avatar-dialog.tsx

```
@@ -127,6 +126,7 @@ export function CreateAvatarDialog({
       }
     },
     onSuccess: () => {
+      debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)